### ISSUE TITLE
Not enough USB WiFi Adapters

### ISSUE THREAD

**Issue by MatthewCroughan** [DoESLiverpool/somebody-should#1497]
Everyone always silently fights over the USB WiFi adapters we have available, so I bought 4 of the worst ones I could find on eBay for £2.99 each. If they're even slightly usable, then they will provide all the remaining non-wired systems with network access. Could solve #1336 

![image](https://user-images.githubusercontent.com/26458780/93708300-f2de2900-fb2c-11ea-88a6-bae0b3a2ebe2.png)

**Comment by MatthewCroughan** [comment]
They have arrived. Testing now.

![image](https://user-images.githubusercontent.com/26458780/94009170-02af7480-fd9c-11ea-94b8-2d779ffa7a49.png)

**Comment by MatthewCroughan** [comment]
The drivers for these devices are not able to be downloaded by Windows 7 since Windows 7 is end of life, and as such the automatic driver downloading service no longer works. Additionally, the signal of the WiFi is too weak in the laser cutting room to provide Gerald's machine with any decent connectivity.

I provided the Ubuntu Vinylcutter PC with one of these adapters and it works fine.

I found a TP Link WiFi dongle to provide Gerald's PC with some sort of connectivity, it is able to sustain 2mbps with a lot of packet loss. The packet loss means that the samba share performs incredibly poorly. It performs better than the adapter that was already present.

All in all, pretty ineffective since the WiFI connectivity in the laser room remains pretty poor.

I will keep the remaining 3 adapters in my drawer, if anyone wants one just ask.
